view-admin/src/components/common/JudgementModal/JudgementModal.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from "react";
+import styles from "./JudgementModal.module.css";
+import { RxCrossCircled } from "react-icons/rx";
+
+interface ModalProps {
+  isOpened: boolean;
+  isIncluded: boolean;
+  canCloseByClickingBackground?: boolean;
+  setisOpened: (isOpened: boolean) => void;
+  setIsIncluded: (included: boolean) => void;
+  setInputNumbers: (numbers: number[]) => void;
+  checkInclusion: () => void;
+}
+
+const Modal = ({
+  isOpened,
+  isIncluded,
+  canCloseByClickingBackground = true,
+  setisOpened,
+  setIsIncluded,
+  setInputNumbers,
+  checkInclusion,
+}: ModalProps) => {
+  const [inputValues, setInputValues] = useState<number[]>([0, 0, 0, 0, 0]);
```

Comment:
```suggestion
  const [inputValues, setInputValues] = useState<number[]>([100, 100, 100, 100, 100]);
```
ここ0は危ないのでありえない数字に置き換えしよう

Reply:
対応しました。